app/views/hyrax/dashboard/collections/_form_share.html.erb

```diff
@@ -1,91 +1,90 @@
-<%# OVERRIDE Hyrax v3.4.2 Use Hyrax::Groups for groups select box %>
+<%# OVERRIDE Hyrax v5.0.0rc2 Use Hyrax::Groups for groups select box %>
 <% access_options = options_for_select([
-    [t('hyrax.dashboard.collections.form_share.access_options.manager'), 'manage'],
-    [t('hyrax.dashboard.collections.form_share.access_options.depositor'), 'deposit'],
-    [t('hyrax.dashboard.collections.form_share.access_options.viewer'), 'view']]) %>
+  [t('hyrax.dashboard.collections.form_share.access_options.manager'), t('manage')],
+  [t('hyrax.dashboard.collections.form_share.access_options.depositor'), t('deposit')],
+  [t('hyrax.dashboard.collections.form_share.access_options.viewer'), t('view')]]) %>
 
 <div id="participants" class="tab-pane">
-  <div class="panel panel-default labels edit-sharing-tab">
-    <div class="panel-body">
+  <div class="edit-sharing-tab">
+    <section class="section-add-sharing">
+      <p><%= t('.note') %></p>
+      <h3><%= t('.add_sharing') %></h3>
 
-      <section class="section-add-sharing">
-        <p><%= t('.note') %></p>
-        <h3><%= t('.add_sharing') %></h3>
+      <!-- Add group form -->
+      <div class="form-add-sharing-wrapper" data-id="<%= @form.id %>">
+        <%= simple_form_for collection_permission_template_form_for(form: @form),
```

Comment:
See above note about `simple_form_for` and `collection_permission_template_form_for`.

Reply:
Looking at Hyrax, this is the correct method name (although idiomatically is perhaps confusingly named named)